Moderation_Matrix.md

```diff
@@ -0,0 +1,59 @@
+# Moderation Matrix
+This matrix correlates with the [Community Rules](https://www.theodinproject.com/community_rules) that are required to be followed in order to participate in the community. Some rules have different levels of severity, so you may see references to them under different numbers of zaps. 
+
+This matrix is not all encompassing and zaps are given at the discretion of the moderator team.
+
+## 0 Zaps (Heads Up)
+- Dog piling/short circuiting a teaching moment
+- Asking about or encouraging the use of Windows
+- "Chat bombing", aka posting many short (ususally 4-5 or fewer words) messages in rapid succession, repeatedly sending parts of thoughts (incomplete, non-standalone sentences or fragments) in new messages. (Use shift + enter and send ONE message where possible or edit your message) 
+- Discussing mental health issues in depth
+
+
+## 1 Zap (WARNING) 
+Minor violations such as:
+- "Mini-modding"
+  - Please leave moderation action to the moderators, use ModMail to report rule breaking or if you see something you’re unsure about in the community
+- Asking about topics or projects unrelated to The Odin Project 
+-Mild unprofessionalism:
+  - Drug related conversation 
+  - Treating others on the server disrespectfully
+  - Mental health issues discussion 
+  - Relationship advice
+- A second infraction of any 0 Zap issues above
```

Comment:
This looks super confusing in the preview, but it just moves the "second infraction of 0 zap" thing to the top. I think it reads better, especially because those are still fresh in the reader's mind.
```suggestion
Minor violations such as:
- A second infraction of any 0 Zap issues above
- "Mini-modding"
  - Please leave moderation action to the moderators, use ModMail to report rule breaking or if you see something you’re unsure about in the community
- Asking about topics or projects unrelated to The Odin Project 
- Mild unprofessionalism:
  - Drug related conversation 
  - Treating others on the server disrespectfully
  - Mental health issues discussion 
  - Relationship advice
```